### ISSUE TITLE
wineasio debugging tips

### ISSUE THREAD

**Issue by Beanow** [theNizo/linux_rocksmith#16]
Getting wineasio set up correctly was a bit fickle for me.

I'd suggest:
- Linking https://forum.vb-audio.com/viewtopic.php?t=1204 as a handy debugger

And making sure you get `WINEPREFIX=$pfx "$proton/bin/wine" ~/Downloads/VBASIOTest32.exe` working.
In my case it was both the realtime permissions not working yet,
and needing to redo `WINEPREFIX=$pfx wine64 regsvr32 wineasio.dll`.

**Comment by theNizo** [comment]
Having a quick look at it, I think any other application that uses ASIO would be fine as well. That said, having a specific one would certainly be good.

> And making sure you get WINEPREFIX=$pfx "$proton/bin/wine" ~/Downloads/VBASIOTest32.exe working.

I don't quite understand what you mean here. I mean, I'd list commands for sure (well, actually I might not have thought about it, thanks), but do you meant to say anything else?

**Comment by Beanow** [comment]
Sure :]
You're right any other ASIO program would work as a test.
Including a modded Rocksmith of course, so if it all just works there's no need.

But I found this one more useful for validating/troubleshooting than using something like a DAW, because:
1. It's light and portable, so it starts up fast and doesn't pollute your prefixes with installs or dependencies.
2. Unlike a 'real world' program has an obvious 32bit and 64bit version test.
3. It's purpose built to enumerate devices and offers a report of what the device looks like through wineasio.

Finally this is less strong of a reason, but this application is more obscure than the well known audio tools like FL or Reaper. I only found this by scavenging the wineasio repository for ideas on what my problem could be. So it would be a "tool you might not have known about" kind of tip too.
